docs/modules/ROOT/pages/serverless-cluster.adoc

```diff
@@ -34,8 +34,9 @@ Development clusters have the following limitations:
 - No cluster scaling
 - No data backups
 - No cluster data persistence
+- Paused after 6 hours without development activity
```

Comment:
Perhaps this is not necessary. Both development and production clusters are idled after 24 hours if there is no payment method.

Reply:
Updated as requested.